pkg/kv/migration.go

```diff
@@ -0,0 +1,61 @@
+package kv
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	kvparams "github.com/treeverse/lakefs/pkg/kv/params"
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+type Migrator interface {
+	Migrate(ctx context.Context) error
+}
+
+type DatabaseMigrator struct {
+	params kvparams.KV
+}
+
+func NewDatabaseMigrator(params kvparams.KV) *DatabaseMigrator {
+	return &DatabaseMigrator{
+		params: params,
+	}
+}
+
+// Migrate TODO (niro): Currently just set up the KV DB Schema version. Need to create a migration flow for KV
```

Comment:
I don't understand what migration is supposed to be run here.

Reply:
Currently I'm keeping the same conventions as in DB. When running setup - it runs the migration flow to reach the most updated DB schema (even though the tables are empty). If and when we reach the point where KV migrations are needed we need to revisit this code and decide how to handle it.